views/layouts/main.handlebars

```diff
@@ -41,7 +41,7 @@
 		
 	{{!-- include other javascript --}}
 	{{#each js}}
-		<script type="module" src="/javascript/{{this}}"></script>
+		<script src="/javascript/{{this}}"></script>
```

Comment:
Don't quite understand what's going on here but I have never seen the "type" field set to anything other than "text/javascript" or something to that effect in my experience. I didn't know you could omit the field, but if it works it works.

Reply:
I had set the type="module" to try and solve an error when importing something in the client side. I didn't end up being necessary, but i'll add the type="text/javascript" back.